color-eyre/tests/theme.rs

```diff
@@ -124,6 +124,8 @@ fn test_panic_backwards_compatibility() {
         .args(["run", "--example", "theme_test_helper"])
         .arg("--no-default-features")
         .args(&features)
+        .env("RUSTFLAGS", "-Awarnings")
+        .env("CARGO_FUTURE_INCOMPAT_REPORT_FREQUENCY", "never")
```

Comment:
This removes cargo printing an extra line on windows warning about a future incompatability in nom.

The cargo output is included in the panic test case as it invokes cargo run and looks at the output.

Since the report is colored with ansi-codes, it gets included in the test check

Reply:
```
    Finished dev [unoptimized + debuginfo] target(s) in 0.03s
     Running `target/debug/examples/theme_test_helper`
[31mThe application panicked (crashed).[0m
Message:  [36m<non string panic payload>[0m
Location: [35mexamples/theme_test_helper.rs[0m:[35m37[0m

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ SPANTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

   0: [91mtheme_test_helper[0m[91m::[0m[91mget_error[0m with [96m[3mmsg[0m[2m=[0m"test"[0m
      at [35mexamples/theme_test_helper.rs[0m:[35m34[0m

  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ BACKTRACE ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
  [96m                              ⋮ 6 frames hidden ⋮                               [0m
   7: [32mstd::panic::panic_any[0m[90m::hd76a7f826307234c[0m
      at [35m/rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/std/src/panic.rs[0m:[35m57[0m
   8: [91mtheme_test_helper::main[0m[90m::h767d3fd6c45048c8[0m
      at [35m/home/jlusby/git/yaahc/color-eyre/examples/theme_test_helper.rs[0m:[35m37[0m
   9: [32mcore::ops::function::FnOnce::call_once[0m[90m::hc5a1cd4127189dad[0m
      at [35m/rustc/f1edd0429582dd29cccacaf50fd134b05593bd9c/library/core/src/ops/function.rs[0m:[35m227[0m
  [96m                              ⋮ 15 frames hidden ⋮                              [0m

Run with COLORBT_SHOW_HIDDEN=1 environment variable to disable frame filtering.
Run with RUST_BACKTRACE=full to include source snippets.
```

This is what we are checking against, note how it included the cargo output as well as the running output.

This has been an issue in the past wrt existing warnings, which was fixed by fixing the warnings.

*This* warning will fix itself in nom and we can not currently do anything about the warning